xarray/core/groupby.py

```diff
@@ -226,7 +226,7 @@ def attrs(self) -> dict:
 
     def __getitem__(self, key):
         if isinstance(key, tuple):
-            key = key[0]
+            (key,) = key
```

Comment:
Was it intentional that multiple values in the tuple will now fail?
```python
(key,) = ("x", "y")
Traceback (most recent call last):

  Cell In[29], line 1
    (key,) = ("x", "y")

ValueError: too many values to unpack (expected 1)
```

Reply:
Yes this is always 1D so should be OK.